android-components/components/support/ktx/src/main/java/mozilla/components/support/ktx/android/view/Activity.kt

```diff
@@ -16,7 +17,8 @@ import androidx.core.view.WindowInsetsControllerCompat
 import mozilla.components.support.base.log.logger.Logger
 
 /**
- * Attempts to enter immersive mode - fullscreen with the status bar and navigation buttons hidden.
+ * Attempts to enter immersive mode - fullscreen with the status bar and navigation buttons hidden,
+ * expending itself into the notch area for devices running Android 28+.
```

Comment:
```suggestion
 * expending itself into the notch area for devices running API 28+.
```
or 
```suggestion
 * expending itself into the notch area for devices running Android version 28+.
```

Preferably suggestion 1 as it's consistent with the below reference to API 30+